lib/rails-settings-ui.rb

```diff
@@ -24,9 +24,12 @@ module RailsSettingsUi
   mattr_accessor :defaults_for_settings
   self.defaults_for_settings = {}
 
+  mattr_accessor :engine_name
+  self.engine_name = "main_app"
+
   class << self
-    def inline_main_app_routes!
-      ::RailsSettingsUi::ApplicationController.helper ::RailsSettingsUi::MainAppRouteDelegator
+    def inline_engine_routes!
```

Comment:
`inline_main_app_routes!` should stay with deprecation message: `warn "[DEPRECATION] `inline_main_app_routes!` is deprecated.  Please use `inline_engine_routes!` instead."`